### DISCUSSION TITLE
Welcome to ZubHub Discussions!

### DISCUSSION THREAD

**Discussion by tuxology** [unstructuredstudio/zubhub#308] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kunalprakash1309** [comment]
Hello Everyone. I hope everyone is fine. Myself Kunal Prakash. My initial application was selected and I am interested in contributing to "Allow creation of learning activities on ZubHub web app" and "Make enhancements to user profiles".
